api/chat.py

```
@@ -161,7 +161,27 @@ def _handle_post(self):
+                if chunks:
+                    context_block = "\n\n".join(
+                        f"[{c['source']} / {c['heading']}]\n{c['content']}"
+                        for c in chunks
+                    )
+                    system_prompt = (
+                        _SYSTEM_PROMPT
+                        + "\n\n---\n\nRELEVANT DOCUMENTATION\n\n"
+                        + context_block
+                    )
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)